app/containers/HomePage/About.js

```diff
@@ -0,0 +1,357 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import * as img1 from 'images/Ico_01.svg';
+import * as img2 from 'images/Ico_02.svg';
+import * as img3 from 'images/Ico_03.svg';
+
+import * as img11 from 'images/1.png';
+import * as img22 from 'images/2.png';
+import * as img33 from 'images/3.png';
+import * as img44 from 'images/4.png';
+import * as img55 from 'images/5.png';
+
+import { SECOND_SCREEN, ANIMATE_IMAGE } from './constants';
+import messages from './messages';
+
+const Box = styled.section`
```

Comment:
I think it may be helpful to create shared styled component for section and define base styles on it:
top padding, bottom padding, etc. and use it for all of the sections.